exporter/opentelemetry-exporter-prometheus-remote-write/tests/test_prometheus_remote_write_exporter.py

```diff
@@ -0,0 +1,377 @@
+        last_value_record.aggregator.update(1)
+        last_value_record.aggregator.update(5)
+        last_value_record.aggregator.take_checkpoint()
+
+        self._exporter.create_timeseries = self._converter_mock()
+        timeseries = self._exporter.convert_from_last_value(last_value_record)
+        self.assertEqual(timeseries[0], (LastValueAggregator, "testname", 5))
+
+    # Ensures value observer aggregator is correctly converted to timeseries
+    def test_convert_from_value_observer(self):
+        value_observer_record = self._generate_record(ValueObserverAggregator)
+        value_observer_record.aggregator.update(5)
+        value_observer_record.aggregator.update(1)
+        value_observer_record.aggregator.update(2)
+        value_observer_record.aggregator.take_checkpoint()
+
+        self._exporter.create_timeseries = self._converter_mock()
+        timeseries = self._exporter.convert_from_value_observer(
+            value_observer_record
+        )
+        self.assertEqual(
+            timeseries[0], (ValueObserverAggregator, "testname_min", 1)
+        )
+        self.assertEqual(
+            timeseries[1], (ValueObserverAggregator, "testname_max", 5)
+        )
+        self.assertEqual(
+            timeseries[2], (ValueObserverAggregator, "testname_sum", 8)
+        )
+        self.assertEqual(
+            timeseries[3], (ValueObserverAggregator, "testname_count", 3)
+        )
+        self.assertEqual(
+            timeseries[4], (ValueObserverAggregator, "testname_last", 2)
+        )
+
+    # Ensures quantile aggregator is correctly converted to timeseries
+    # TODO: Add test once method is implemented
+    def test_convert_from_quantile(self):
+        pass
+
+    # Ensures timeseries produced contains appropriate sample and labels
+    def test_create_timeseries(self):
+        sum_aggregator = SumAggregator()
+        sum_aggregator.update(5)
+        sum_aggregator.take_checkpoint()
+        sum_aggregator.last_update_timestamp = 10
+        export_record = ExportRecord(
+            self._test_metric,
+            get_dict_as_key({"record_name": "record_value"}),
+            sum_aggregator,
+            Resource({"resource_name": "resource_value"}),
+        )
+
+        expected_timeseries = TimeSeries()
+        expected_timeseries.labels.append(
+            self._exporter.create_label("__name__", "testname")
+        )
+        expected_timeseries.labels.append(
+            self._exporter.create_label("resource_name", "resource_value")
+        )
+        expected_timeseries.labels.append(
+            self._exporter.create_label("record_name", "record_value")
+        )
+        expected_timeseries.samples.append(
+            self._exporter.create_sample(10, 5.0),
+        )
+        timeseries = self._exporter.create_timeseries(
+            export_record, "testname", 5.0,
+        )
+        self.assertEqual(timeseries, expected_timeseries)
+
+
+class ResponseStub:
```

Comment:
I would recommend using `Mock`s as much as possible. This class can be replaced with a dictionary that is later used as the argument to `configure_mock`. Having classes like this one hides the fact that they are only for testing purposes, even when their name attempts to indicate so.

Reply:
Got rid of this Stub to use the Mock class combined with configure_mock as suggested. Thanks for the tip!